src/Helpers/constants.js

```diff
@@ -480,7 +488,7 @@ export const DEFAULT_FILTER_DATA = {
     status_id: [],
     impact: [],
     publish_date: 'all',
-    security_rule: 'all',
+    security_rule: 'true,false',
```

Comment:
do you want to keep `security_rule` instead of `rule_presence`

Reply:
yes, see below